src/app/app.module.ts

```diff
@@ -80,6 +81,8 @@ import {
 } from './components/challenge/challengeviewallsubmissions/challengeviewallsubmissions.component';
 import { SideBarComponent } from './components/utility/side-bar/side-bar.component';
 
+import {MatTableModule} from '@angular/material/table';
```

Comment:
```suggestion
import { MatTableModule } from '@angular/material/table';
```
Uniform spacing. Also for the line which is below.

Reply:
Done